chia/wallet/cat_wallet/cat_wallet.py

```diff
@@ -208,6 +208,12 @@ async def get_or_create_wallet_for_cat(
 
         self.lineage_store = await CATLineageStore.create(self.wallet_state_manager.db_wrapper, self.get_asset_id())
         await self.wallet_state_manager.add_new_wallet(self, self.id())
+
+        for state in await self.wallet_state_manager.interested_store.get_unacknowledged_states_for_asset_id(
+            limitations_program_hash, delete_after=True
+        ):
+            await self.wallet_state_manager.retry_store.add_state(*state)
```

Comment:
so this means we store coin states for CATs that we didn't accept, and once accepted we replay them to get them added to the wallet?

Reply:
That's right